frontend/packages/dev-console/src/components/pipelines/__tests__/Pipeline.spec.tsx

```diff
@@ -18,11 +19,29 @@ const pipelinePageProps = {
   },
 };
 
-const pipelineWrapper = shallow(<PipelinesPage {...pipelinePageProps} />);
+const pipelinePagePropsWNS = {
+  history: null,
+  location: null,
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+    url: 'k8s/ns/my-project/tekton.dev~v1alpha1~Pipeline',
+    params: {
+      ns: 'all-projects',
+    },
+  },
+};
 
 describe('Pipeline List', () => {
+  const pipelineWrapperNS = shallow(<PipelinesPage {...pipelinePageProps} />);
   it('Renders a list', () => {
-    expect(pipelineWrapper.exists()).toBe(true);
-    expect(pipelineWrapper.find(ListPage).exists());
+    expect(pipelineWrapperNS.exists()).toBe(true);
+    expect(pipelineWrapperNS.find(ListPage).exists());
```

Comment:
this statement is not complete plz add the expected behaviour.

`expect(pipelineWrapperNS.find(ListPage).exists()).toBeTruthy()`

Reply:
`ListPage` will not be found! you need to assert presence of `PipelinesResourceList`.

Reply:
@sahil143 Updated to check for value true